### PR TITLE
Updated service account links

### PR DESCRIPTION
**Pull request by jerae-duffin**
Add Service Account note and links to Registry. Added Service Account link to Docker Hub/Download Rate Limits.

### CODE REVIEW

**Review by jeremy-castile — APPROVED** [review]
LGTM

File: registry/recipes/mirror.md [hunk 1]
```diff
@@ -76,6 +76,10 @@ Multiple registry caches can be deployed over the same back-end. A single
 registry cache ensures that concurrent requests do not pull duplicate data,
 but this property does not hold true for a registry cache cluster.
 
+**Note**
+>
+> Service accounts included in the Team plan are limited to 15,000 pulls per day. See [Service Accounts](https://docs.docker.com/docker-hub/service-accounts/){: target="blank" rel="noopener" class=“”} for more details.
```
Comment (usha-mandya) [inline registry/recipes/mirror.md:81]: @jerae-duffin Could you add a relative path here instead of the full URL?

**Comment by jerae-duffin** [comment]
@usha-mandya  updated links ready for review

**Review by usha-mandya — APPROVED** [review]
LGTM

**Pull request merged**